services/cd-service/pkg/cmd/server.go

```diff
@@ -260,7 +270,7 @@ func RunServer() {
 				KnownHostsFile: c.GitSshKnownHosts,
 			},
 			Branch:                 c.GitBranch,
-			GcFrequency:            20,
+			GcFrequency:            c.ReleaseVersionsLimit,
```

Comment:
this is the wrong field.
See `keptVersionsOnCleanup` in transformer.go

Reply:
Done